### DISCUSSION TITLE
Feedback on documentation

### DISCUSSION THREAD

**Discussion by smitdave** [dd-harp/ramp.xds#111] [General]
I've been writing documentation, but I'm not sure how much of it is really useful. 

To be clear, I'm writing the documentation partly to check that the software does what I want it to do. The process of writing out a description enforces a kind of discipline. 

Can I ask for feedback on the documentation that I have written? 

@aucarter @doreenmbabazi @Jnsumba

**Comment by smitdave** [comment]
Examples: 

- [dLdt.basicL](https://dd-harp.github.io/ramp.xds/reference/dLdt.basicL.html) 
- [make_xds_object](https://dd-harp.github.io/ramp.xds/reference/make_xds_object.html)
- [setup_EGG_LAYING](https://dd-harp.github.io/ramp.xds/reference/setup_EGG_LAYING.html)

**Comment by smitdave** [comment]
I'm looking for general comments and opinions. Is this too much, too little?

**Reply by aucarter** [reply]
Not too little... I like having things laid out clearly in the function descriptions along with citations. This seems like a very good template. Side note, some arguments seem to differ between Arguments and Description in `make_xds_object`.
<img width="521" alt="image" src="https://github.com/user-attachments/assets/83816550-cd50-45d6-8e36-fac7138a5d71">
